domain/objectstore/state/state.go

```diff
@@ -221,9 +221,10 @@ AND size = $dbMetadata.size`, dbMetadata, dbMetadataPath)
 			return errors.Errorf("inserting metadata: %w", err)
 		}
 
-		if rows, err := outcome.Result().RowsAffected(); err != nil {
+		metadataRows, err := outcome.Result().RowsAffected()
+		if err != nil {
 			return errors.Errorf("inserting metadata: %w", err)
-		} else if rows != 1 {
+		} else if metadataRows != 1 {
```

Comment:
Isn't it better to just check if it's exactly 0?
```suggestion
		} else if metadataRows == 0 {
```

Reply:
Yeah, no idea why it was `!= 1`.

Reply:
Done.